### DISCUSSION TITLE
Can anyone tell me what to put in code and codeVerifier in oAuth2.0

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#803] [Q&A] [answered]
Hey! I am trying to setup login via GitHub, I got the clientId and clientSecret and put them in pb admin and activated. But I am not understanding what to send in `code` and `codeVerifier` please let me know.

**Comment by ganigeorgiev** [comment, marked as answer]
@rajesh6161 You can follow the "Auth via OAuth2" example from https://pocketbase.io/docs/manage-users/#auth-via-oauth2.
